src/govuk/components/character-count/character-count.unit.test.mjs

```diff
@@ -0,0 +1,148 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { createElement } from '../../../../lib/dom-helpers.mjs'
+import CharacterCount from './character-count.mjs'
+
+describe('CharacterCount', () => {
+  describe('formatCountMessage', () => {
+    describe('default configuration', () => {
+      let component
+      beforeAll(() => {
+        // The component won't initialise if we don't pass it an element
+        component = new CharacterCount(createElement('div'))
+      })
+
+      it('formats singular remaining characters', () => {
```

Comment:
This might be [a good use case for `it.each`](https://elfi-y.medium.com/reduce-boilerplate-test-code-with-jest-it-each-30a0eec9776d) as it's quite repetitive ('given this number of remaining characters, the message should be this')